src/xpk/templates/fs-pv.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: v1
+kind: PersistentVolume
+metadata:
+  name: xpk-filestore-pv
+spec:
+  storageClassName:
+  capacity:
+    storage: 
+  accessModes:
+  persistentVolumeReclaimPolicy: Retain
+  volumeMode: Filesystem
+  csi:
+    driver: filestore.csi.storage.gke.io
+    volumeHandle:
+    volumeAttributes:
+      ip:
+      volume:
```

Comment:
I would suggest to rename the file to filestore-pv.yaml to make its name to be a more self explanatory. fs is s bit too generic as a name prefix.